7.1 Snapshot.md

```diff
@@ -0,0 +1,53 @@
++ Approval weight: We require that the approval weight is concluded at a time much before messages reach the pruning age.
++ Voting: We require that any voting is concluded much before messages reach the pruning age. 
++ Data flow: A weak past cone check is applied.
+
+
+## 7.1.3 Proof of Inclusion and Pruning
+
+In order to keep the ledger concise, data that is sufficiently old can be removed. We call this process pruning. Thus, the IOTA 2.0 protocol for a standard node does not attempt to keep all information back until the Genesis. Rather the protocol effectively becomes a mechanism to prove whether a certain message was, at some point in the past, included in the ledger. This can be done e.g. through a Merkle tree proof. 
+
+Transactions are payloads of value messages, that cannot be pruned. Specifically when a transaction contains an unspent output, see [5.1. UTXO](./5.1%20UTXO.md), the transaction shall be kept when pruning the ledger. An optimization of this process is possible by reducing the message to only keeping the information about the unspent outputs, however the implementation will be more complicated, and thus is not considered for this first iteration of the protocol.
+
+## 7.1.5 Pruning dependencies
+
+We require several conditions to be fulfilled in order to allow for a smooth pruning process. Pruning is done when messages reach the age `DELTA_PRUNE`. 
+
+Since messages cannot attach to a message whose timestamp has an age difference of more than `DELTA`, see [4.3.4 R-URTS](./4.3%20Tip%20Selection%20Algorithm.md/#4.3.4%20R-URTS), and the timestamp precision is enforced through voting, see [4.2.3 General Timestamp rules](./4.2%20Timestamps.md/#4.2.3%20General%20Timestamp%20rules). 
+Finalization of messages must be guaranteed to happen before the point of pruning. Thus, any message must be either confirmed or rejected before the elapse of the pruning age. This requires that both the filtering out of messages with wrong timestamps, voting on conflicts, as well as the accumulation of sufficient approval weight takes a time much smaller than `DELTA_PRUNE`. 
+
+In order to allow a smooth pruning, any message at the age `DELTA_PRUNE` must either belong to the branch `masterBranch` or to a rejected branch. This simply means that messages are either confirmed or orphaned. 
+
+Transaction must pass a weak past cone check, see [2.4.4.6 Booker - Point 7](./2.4%20Data%20Flow.md/#2.4.4.6%20Booker). 
+
+With the above conditions we can safely assume any message before the time `DELTA_PRUNE>>DELTA` can be pruned. 
+
+## 7.1.6 Pruning process
+
+An output is considered *confirmed* spent, when the message containing a transaction spending it is confirmed. A flag `confirmedSpent` *shall* be set to `TRUE` in the metadata of the output when this event is triggered.
+
+Once a message reaches the age `DELTA_PRUNE` several checks are performed:
+
++ If the message is a data message it can be removed. 
```

Comment:
Making this change here since we never were clear that our many messages types, as the ones used in dRNG or FPC statements, are data messages.
```suggestion
+ If the message is not a value message it can be removed. 
```